### ISSUE TITLE
Questions about the evaluation

### ISSUE THREAD

**Issue by qianqianwang68** [deepmind/tapnet#9]
Hi TapNet authors,

Just to confirm if I understand the evaluation correctly: It is clear that you train and test tap-net on resized images of 256x256.  When you run the baseline methods, you also use resized images of 256x256 as input, is that correct?

If so, could you explain why it is preferable to evaluate on reduced resolution when the full-resolution images & annotations are available?

**Comment by cdoersch** [comment]
Hi @qianqianwang68,

You are correct that we resized our videos to 256x256.  Regarding higher resolution, the short answer is that you are allowed to use full resolution at test time, but we consider this to be a **separate evaluation**.  If you have a method which can handle high-resolution videos, feel free to use it.  Just make sure that you downscale your predictions and the ground truth to 256x256 before reporting numbers.

Because it's strictly easier to evaluate using a higher resolution, we consider any method that runs on 256x256 to be a competitor on both versions of the benchmark.  However, methods which were run on higher resolutions to be only valid competitors on the "any resolution" version.  Therefore, TAP-Net is currently the SOTA on both benchmarks, but it's possible that different algorithms will be SOTA on the two versions in the future.

There are two reasons we decided to do it this way.  The first is that it's computationally expensive to work on high resolution, and not all institutions have the resources to do this; we want to give an opportunity to as many people as possible.  Second, we can get more accurate ground truth if we label the images at a higher resolution than the algorithms get to see.  At 1080p resolution, you'll likely notice some drift.  We don't want to give an andvantage to algorithms that replicate human errors, so we like to minimize the errors in our dataset as much as possible.  Ideally we would have a 720p benchmark which is labeled at 4K resolution, but this isn't feasible with our current setup.  If there's demand, we may implement this in the future.

Hope that helps, and I look forward to seeing your work on our dataset!
